### PR TITLE
add non-accurate distance to feed profile view

### PR DESCRIPTION
**Pull request by akhrail1996**
supporting https://github.com/getsince/test5/pull/246

### CODE REVIEW

File: lib/t/feeds.ex [hunk 1]
```diff
@@ -145,6 +147,20 @@ defmodule T.Feeds do
     end
   end
 
+  defp with_distance(query, location) do
+    if location do
+      select_merge(query, [p], %{
+        distance:
+          fragment(
+            "ST_DistanceSphere(?::geometry, location::geometry)::integer / 1000",
```
Comment (akhrail1996) [inline lib/t/feeds.ex:155]: ::integer + integer division ensures non-accuracy of distance (impossible to triangulate the exact location)

**Comment by ruslandoga** [comment]
@akhrail1996 maybe something like https://github.com/getsince/test3/commit/974162afa4b740590d857fa6a13e08b5eac76e66?

**Comment by ruslandoga** [comment]
[Comments](https://old.reddit.com/r/netsec/comments/1ydao6/how_i_was_able_to_track_the_location_of_any/) and the [article itself](https://blog.includesecurity.com/2014/02/how-i-was-able-to-track-the-location-of-any-tinder-user/) seem to show that even rounding in the filter still gives off the information about the real user location with enough data points.

File: lib/t/feeds.ex [hunk 2]
```diff
@@ -152,7 +152,7 @@ defmodule T.Feeds do
       select_merge(query, [p], %{
         distance:
           fragment(
-            "ST_DistanceSphere(?::geometry, location::geometry)::integer / 1000",
+            "round(ST_DistanceSphere(ST_ReducePrecision(?::geometry, 0.01), ST_ReducePrecision(location::geometry, 0.01)) / 1000)::integer",
```
Comment (akhrail1996) [inline lib/t/feeds.ex:155]: @ruslandoga how about using reduced precision?

**Comment by akhrail1996** [comment]
@ruslandoga it works locally I don't understand why this error occurs here:
image
while data types are correct:
image
https://postgis.net/docs/ST_ReducePrecision.html

**Review by ruslandoga — APPROVED** [review]

**Review by ruslandoga — APPROVED** [review]

**Pull request merged**